packages/modelviewer.dev/styles/docs.css

```diff
@@ -190,3 +172,35 @@ hr.solid {
 .bolden {
   font-weight: bold;
 }
+
+/* Smaller screen/ mobile */
+@media only screen and (max-width: 800px) {
+  :root {
+    --sidebar-width: 0;
+  }
+
+  .sidebar {
+    width: var(--sidebar-width);
+  }
+
+  .outer-content {
+    margin-left: 0;
```

Comment:
The sidebar toggle should probably only show on mobile and should only show on the header. It's a little strange that you can have two of them on the page at once when you're near another section. Maybe push it over to the left margin and help it look more like a button.

Reply:
yeah that's a good call